sdks/java/io/jdbc/src/main/java/org/apache/beam/sdk/io/jdbc/JdbcUtil.java

```diff
@@ -0,0 +1,104 @@
+package org.apache.beam.sdk.io.jdbc;
+
+import org.apache.beam.sdk.schemas.Schema;
+import org.apache.beam.sdk.values.Row;
+
+import java.sql.Date;
+import java.sql.JDBCType;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.sql.Time;
+import java.sql.Timestamp;
+import java.util.List;
+import java.util.stream.IntStream;
+
+public class JdbcUtil {
+
+    public static void mapDataAccordingToSchema(Row element, PreparedStatement preparedStatement, Schema.FieldType fieldType, int index) {
```

Comment:
There's already a `SchemaUtil` class. I think a separate `JdbcUtil` class is unnecessary and confusing because it is still dealing with schemas anyway.

Also, if you look at how the read path is implemented using `ResultSetFieldExtractors`, the same pattern can be applied to setting the parameters of a prepared statement. It's better to be consistent with the existing code base so I suggest reworking this code to match.

Reply:
We have added JdbcUtil class because schema class is in your 6674 branch and its not merged into ours. We are planning to merge your work in our branch once it is approved and merged in the master branch otherwise reviewing the PR will be difficult.

Reply:
I have reworked the setting of data in prepared statement. Please have a look